armi/nuclearDataIO/cccc.py

```diff
@@ -689,6 +706,57 @@ def _write(cls, lib, fileName, fileMode):
         raise NotImplementedError()
 
 
+class DataContainer:
+    """
+    Data representation that can be read/written to/from with a Stream.
```

Comment:
from a `CCCC.Stream`. We are overloading a very meaningful term here with Stream, so should be careful.